static_frame/core/frame.py

```diff
@@ -1975,6 +1975,12 @@ def _iter_element_loc(self):
     #---------------------------------------------------------------------------
     # transformations resulting in the same dimensionality
 
+    def __reversed__(self) -> tp.Iterable[np.ndarray]:
+        '''
+        Returns a reverse iterator on the frame's rows.
+        '''
+        return reversed(self.values)
```

Comment:
In both of these cases, I think we want to iterate over the index, like a dictionary. Basically, a reversed version of `iter(frame)` or `iter(series)`. So this would give the desired behavior:

```suggestion
        return reversed(self.index)
```